### PR TITLE
Mlc 392 pps improvements

### PR DESCRIPTION
**Pull request by nirhutnik**

### CODE REVIEW

File: deepchecks/checks/methodology/single_feature_contribution_train_test.py [hunk 1]
```diff
@@ -129,3 +172,33 @@ def condition(value: t.Dict[Hashable, float]) -> ConditionResult:
         return \
             self.add_condition(f'Train-Test features\' {pps_html_url} (PPS) difference is not greater than {threshold}',
                                condition)
+
+    def add_condition_feature_pps_in_train_not_greater_than(self: FC, threshold: float = 0.2) -> FC:
```
Comment (noamzbr) [inline deepchecks/checks/methodology/single_feature_contribution_train_test.py:176]:
```suggestion
    def add_condition_feature_pps_in_train_not_greater_than(self: FC, threshold: float = 0.85) -> FC:
```

File: deepchecks/checks/methodology/single_feature_contribution_train_test.py [hunk 2]
```diff
@@ -129,3 +172,33 @@ def condition(value: t.Dict[Hashable, float]) -> ConditionResult:
...
+                if pps_value > threshold
+            ]
+
+            if failed_features:
+                message = f'Features in train dataset with PPS above threshold: {", ".join(map(str, failed_features))}'
+                return ConditionResult(False, message)
+            else:
+                return ConditionResult(True)
+
+        pps_url = 'https://towardsdatascience.com/rip-correlation-introducing-the-predictive-power-score-3d90808b9598'
```
Comment (noamzbr) [inline deepchecks/checks/methodology/single_feature_contribution_train_test.py:198]: Can we change this to a tinyurl?

**Review by noamzbr — CHANGES_REQUESTED** [review]

File: deepchecks/checks/methodology/single_feature_contribution_train_test.py (hunk 2 above)
Comment (nirhutnik) [reply deepchecks/checks/methodology/single_feature_contribution_train_test.py:198]: done

File: deepchecks/checks/methodology/single_feature_contribution_train_test.py (hunk 1 above)
Comment (nirhutnik) [reply deepchecks/checks/methodology/single_feature_contribution_train_test.py:176]: I think that's too high. Changing to 0.7, ok?

**Review by noamzbr — APPROVED** [review]
Approved, pending clarification that the pps score is feature -> label

**Comment by nirhutnik** [comment]
Closes #353

**Pull request merged**